acceptance-tests/features/add_title_number_to_a_case.feature

```diff
@@ -0,0 +1,32 @@
+@add_title_number_to_a_case
```

Comment:
Fair point and I've added an extra tag for user story number now so people can run by specifying either @USXX or @name_of_user_story